### PR TITLE
Small syndicate update

### PR DESCRIPTION
**Pull request by NULL882**
## About the PR
This PR adds new things to the syndicate. Including the doors for the Nukeops shuttle.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

**Changelog**

:cl: @NULL882
- add: Added comander syndicate PDA.
- add: Syndicate airlock with Nukeops access.
- add: Box of poison for jobs: Botanist, Bartender, Chef, MedicalDoctor, Chemist in Uplink.
- add: Mechanical tool box with insulated gloves for jobs: Passenger, Mime, Clown, Botanist, CargoTechnician in Uplink.
- add: Dual energy sword in Uplink.
- add: Fake CentCom official costume in Uplink.
- add: Armory locker for station Armory.
- add: Nuke Operative access for operative closet.
- tweak: Syndicate PDA.
- tweak: Nukeops access.

### CODE REVIEW

**Comment by HoidC** [comment]
sussy toolbox already exists for the mechanical toolbox, dunno if adding "generic item you get on station but in uplink" is a good idea

dunno why we need a double esword when we have a single esword

centcomm official outfit was axed a while ago, forgot why but don't know if that's allowed yet, either way chameleon works for it + people aren't going to roleplay with it anyway and thus just more surplus bloat making it even worse than it already is

**Comment by Emisse** [comment]
good stuff in here but i mostly agree with hoid regarding the toolbox and d blade, d blade is notoriously hard to balance with 2 other energy melee weapons

**Comment by HoidC** [comment]
been brought to my attention that the toxin kit has the regular toxin chemical, histamine and amatoxin which imo are all bad choices

amatoxin is obtainable through normal gameplay anyway, histamine is absolutely worthless and toxin is a debug chemical I think

would be better if it was different poisons that did different things

**Comment by NULL882** [comment]
> been brought to my attention that the toxin kit has the regular toxin chemical, histamine and amatoxin which imo are all bad choices
> 
> amatoxin is obtainable through normal gameplay anyway, histamine is absolutely worthless and toxin is a debug chemical I think
> 
> would be better if it was different poisons that did different things

Hmmm, maybe after the PR merge I'll change the toxins and maybe add a new chemical.

**Comment by NULL882** [comment]
> sussy toolbox already exists for the mechanical toolbox, dunno if adding "generic item you get on station but in uplink" is a good idea
> 
> dunno why we need a double esword when we have a single esword
> 
> centcomm official outfit was axed a while ago, forgot why but don't know if that's allowed yet, either way chameleon works for it + people aren't going to roleplay with it anyway and thus just more surplus bloat making it even worse than it already is

Mechanical tool box for more covert activities of agents, it's just somehow suspection to see a passenger in combat gloves...

**Comment by NULL882** [comment]
Dual esword will be useful for Nyukeops in case of a shortage of cartridge

**Comment by HoidC** [comment]
> Dual esword will be useful for Nukeops in case of a shortage of cartridge

there's a single esword for that

**Comment by Fellowsinspace** [comment]
I mean if they wanted to make the D-sword different they could give it a higher reflect stat and keep the same damage as an E-sword but again that's kinda busted.

**Comment by GoodWheatley** [comment]
I don’t think desword would infringe on esword’s niche too much, but I don’t think the time is right to add it since the reflection system needs adjusting to properly distinguish between ballistics and energy projectiles

**Comment by metalgearsloth** [comment]
cl needs to be human-readable without the icon, "Syndicate airlock" doesn't mean much.

**Comment by NULL882** [comment]
> cl needs to be human-readable without the icon, "Syndicate airlock" doesn't mean much.

Done

**Comment by DjfjdfofdjfjD** [comment]
Why only 14TC? The price of the Esword was raised 6->8 after the addition of Reflection(and other buff), I find it logical that the DUAL Energy Sword is TWO times more expensive than the normal one.

**Comment by Slava0135** [comment]
Double energy sword should reflect 100% of energy shots and 50% of ballistic shots to make it more distinct.

**Comment by HoidC** [comment]
I don’t really get why we need a flat upgrade weapon when we already have a single Esword that deflects bullets and does like 30 damage a swing. We have the fire axe for any other dedicated melee people and the epen as an emergency sidearm. There’s a balance here that’s made and would be interrupted by adding a best option.

**Comment by NULL882** [comment]
> I don’t really get why we need a flat upgrade weapon when we already have a single Esword that deflects bullets and does like 30 damage a swing. We have the fire axe for any other dedicated melee people and the epen as an emergency sidearm. There’s a balance here that’s made and would be interrupted by adding a best option.

Diversify uplink

**Comment by NULL882** [comment]
> Double energy sword should reflect 100% of energy shots and 50% of ballistic shots to make it more distinct.

75% of energy, 50% of ballistic

**Comment by NULL882** [comment]
> Why only 14TC? The price of the Esword was raised 6->8 after the addition of Reflection(and other buff), I find it logical that the DUAL Energy Sword is TWO times more expensive than the normal one.

raised the price to 16

File: Resources/Textures/Objects/Weapons/Melee/e_dual_sword.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    {
+      "name": "inhand-left-blade",
+      "directions": 4,
+      "delays": [
+        [
+          0.1,
+          0.1
+        ],
+		[
+          0.1,
```
Comment (metalgearsloth) [inline Resources/Textures/Objects/Weapons/Melee/e_dual_sword.rsi/meta.json:42]: Don't mix tabs and spaces please.

Comment (NULL882) [reply Resources/Textures/Objects/Weapons/Melee/e_dual_sword.rsi/meta.json:42]: Well, at least it works, I got a little confused when meta.json

**Comment by HoidC** [comment]
> > I don’t really get why we need a flat upgrade weapon when we already have a single Esword that deflects bullets and does like 30 damage a swing. We have the fire axe for any other dedicated melee people and the epen as an emergency sidearm. There’s a balance here that’s made and would be interrupted by adding a best option.
> 
> Diversify uplink

By adding a weapon that combines the best features of the fire axe (structural damage), the best features of the Esword (split damage types and deflecting projectiles) and is also one handed?

**Comment by NULL882** [comment]
> > > Я действительно не понимаю, зачем нам плоское улучшенное оружие, когда у нас уже есть один Esword, который отклоняет пули и наносит около 30 единиц урона за взмах. У нас есть пожарный топор для любых других преданных рукопашному бою людей и epen в качестве аварийного оружия. Здесь есть баланс, который достигнут и будет нарушен добавлением наилучшего варианта.
> > 
> > 
> > Диверсифицировать восходящую линию связи
> 
> Добавив оружие, которое сочетает в себе лучшие характеристики пожарного топора (структурный урон), лучшие характеристики Esword (разделенные типы урона и отклоняющие снаряды), а также одноручное?

Dual E sword useful for operatives and very well suited to finish off the target, it also fits in a pocket.

**Comment by NULL882** [comment]
Wha...

**Comment by NULL882** [comment]
> > > I don’t really get why we need a flat upgrade weapon when we already have a single Esword that deflects bullets and does like 30 damage a swing. We have the fire axe for any other dedicated melee people and the epen as an emergency sidearm. There’s a balance here that’s made and would be interrupted by adding a best option.
> > 
> > 
> > Diversify uplink
> 
> By adding a weapon that combines the best features of the fire axe (structural damage), the best features of the Esword (split damage types and deflecting projectiles) and is also one handed?

The double sword is well suited for operatives, and for its full-fledged work you need to take it in two hands.

**Comment by HoidC** [comment]
> > > > I don’t really get why we need a flat upgrade weapon when we already have a single Esword that deflects bullets and does like 30 damage a swing. We have the fire axe for any other dedicated melee people and the epen as an emergency sidearm. There’s a balance here that’s made and would be interrupted by adding a best option.
> > > 
> > > 
> > > Diversify uplink
> > 
> > 
> > By adding a weapon that combines the best features of the fire axe (structural damage), the best features of the Esword (split damage types and deflecting projectiles) and is also one handed?
> 
> The double sword is well suited for operatives, and for its full-fledged work you need to take it in two hands.

Why would you ever buy the fire axe over this

**Comment by FellowsinSpess** [comment]
> > > By adding a weapon that combines the best features of the fire axe (structural damage), the best features of the Esword (split damage types and deflecting projectiles) and is also one handed?
> > 
> > 
> > The double sword is well suited for operatives, and for its full-fledged work you need to take it in two hands.
> 
> Why would you ever buy the fire axe over this

Because it has higher structural? You're comparing the D-Swords 4 Structural damage to the Fire axes 60. I don't see how its going to "outclass" the fire axe  in its destructive capability, if anything its just a damage and reflect upgrade from the E-sword. Not to mention the fire axe is cheaper cost wise.

**Comment by asperger-sind** [comment]
I'm personally of the opinion that your balancing and reasoning for said balancing is odd at best, a 40 (44 with structural) damage, 1 attack speed, togglable weapon with reflects for the price of 16 TC designed for Nukies simply isn't going to be balanced, you can crit a person with a heavy attack and a light attack whilst pulverizing the majority of the armor because of split damage types.

A player barely confident with melee and a mild amount of understanding regarding gamesense, target prioritization and fight selection can buy that, a stim of any kind and just forget that you have a nukie team and 1v95 the station because of how much damage, protection the de-sword will provide, intending it as a sidearm is in my eyes stupid because we already have a singular e-sword which is reasonably balanced and does work with the role well.

The poison kit's chemical selection is not alright whatsoever (histamine, amatoxin, toxin) and the reasoning of "fixing it after merge" doesn't stick with me, I'd honestly just make custom unobtainable chems that you can mix together for different effects and damage categories.

The mechanical toolbox looks like bloat in my eyes because of the fact that we already have the suspicious toolbox, both of them fulfill the same exact role, and I don't agree on adding another toolbox because it's "better for stealth", mind you, it does cost 3 TC, which is already more than the suspicious toolbox with it's vastly more advanced gear.

The QoL stuff (PDAs, access, lockers, airlocks, et cetera) is nice tho, I'm not sure about the airlock sprite it looks too out of place.


tl;dr, balancing bad, go discuss on discord and rebalance imho.

also about the pr itself, tons of typos with "comander" instead of "commander", probably best to say syndie instead of syndi, and generally go with "syndicateCommanderPDA" instead of "syndiComPDA" maybe not, i don't know conventions too well
would do a review but github dies if i try to lmao

**Comment by Emisse** [comment]
i agree with most of the points brought up above

**Comment by NULL882** [comment]
Well, if my PR is inappropriate here then I'll come later ...

**Comment by Emisse** [comment]
id suggest atomizing this PR into smaller sections so we can merge the things that are ready to go and the other things can be addressed separately
